src/main/java/mil/dds/anet/beans/EmailDeactivationWarning.java

```diff
@@ -0,0 +1,64 @@
+package mil.dds.anet.beans;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import io.leangen.graphql.annotations.GraphQLIgnore;
+import java.time.Instant;
+import java.util.Objects;
+import mil.dds.anet.views.AbstractAnetBean;
+
+public class EmailDeactivationWarning extends AbstractAnetBean {
+
+  private ForeignObjectHolder<Person> person = new ForeignObjectHolder<>();
+  private Instant sentAt;
+
+  @JsonIgnore
+  @GraphQLIgnore
```

Comment:
Since this class is not exposed over a GraphQL endpoint, the annotation is not necessary. If you wish, you could also move it to the class level (and remove it from the member level). The `@JsonIgnore` annotation is completely unnecessary, as that is only used by the JSON requests from the GraphQL test client in the server-side tests, and there is no service that uses this class.